addons/binding/org.openhab.binding.solaredge/README.md

```diff
@@ -16,16 +16,18 @@ Auto-Discovery is not supported, as access requires authentication.
 The following configuration parameters are available for this thing:
 
 - **tokenOrApiKey** (required)
-Either the official API Key for using the public API or when using the inofficial private API: a token which can be retrieved from browser's cookie store when logged into the solaredge website. It is called 'SPRING_SECURITY_REMEMBER_ME_COOKIE'
+Either the [official API Key](https://www.youtube.com/watch?v=iR26nmL5bXg) for using the public API or when using the inofficial private API: a token which can be retrieved from browser's cookie store when logged into the solaredge website. It is called 'SPRING_SECURITY_REMEMBER_ME_COOKIE' When using this token, see also usePrivateApi and meterInstalled.
```

Comment:
You would make me happy if you put each sentence in a new line. I will be much better readable for future maintenance.